testsuite/features/support/client_stack.rb

```diff
@@ -50,12 +50,13 @@ def check_restart(host, node, time_out)
 # We get these data decoding the values in '/etc/os-release'
 # rubocop:disable Metrics/AbcSize
 def get_os_version(node)
-  os_family_raw, code = node.run('grep "^ID=" /etc/os-release')
-  raise "No os-release found in the node: #{node.hostname}" unless code.zero?
+  os_family_raw, code = node.run('grep "^ID=" /etc/os-release', false)
+  return nil, nil unless code.zero?
   os_family = os_family_raw.strip.split('=')[1]
   return nil, nil if os_family.nil?
   os_family.delete! '"'
-  os_version_raw = node.run('grep "^VERSION_ID=" /etc/os-release')
+  os_version_raw, code = node.run('grep "^VERSION_ID=" /etc/os-release', false)
+  return nil, nil unless code.zero?
   os_version = os_version_raw.strip.split('=')[1]
   return nil, nil if os_version.nil?
```

Comment:
We could at least return the os_family, which is not nil at that point of the code.
But ok, as we were in a hurry.

Reply:
I don't think it would be useful half the info, either everything or nil